Library/Formula/lnav.rb

```diff
@@ -13,10 +12,18 @@ class Lnav < Formula
     sha256 "26892fd713417b2b84a6615a63d9492a683f53a7fb497dd2b4a681d7267cc2a5" => :mountain_lion
   end
 
+  head do
+      url 'https://github.com/tstack/lnav.git'
```

Comment:
If you could fix the single quotes > double quotes here since we're moving the line anyway, that'd be :heart:.